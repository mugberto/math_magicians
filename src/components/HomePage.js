import React from 'react';

function HomePage() {
  return (
    <React.Fragment>
      <h2>Welecome</h2>
      <p>
        Lorem ipsum dolor, sit amet consectetur adipisicing elit. 
        Doloremque minus similique impedit iste sunt est consequuntur ratione, 
        aperiam nobis quidem. 
        Quo sunt, quia sequi delectus, quaerat ipsa nam hic suscipit. 
      </p>
      <p>
        Lorem ipsum dolor, sit amet consectetur adipisicing elit. 
        Doloremque minus similique impedit iste sunt est consequuntur ratione, 
        aperiam nobis quidem. 
        Quo sunt, quia sequi delectus, quaerat ipsa nam hic suscipit. 
      </p>
    </React.Fragment>
  );
}

export default HomePage;