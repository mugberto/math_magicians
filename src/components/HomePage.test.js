import React from 'react';
import renderer from 'react-test-renderer';
import HomePage from './HomePage';

test('Should render correctly', () => {
  const tree = renderer.create(<HomePage />).toJSON();
  expect(tree).toMatchSnapshot();
});
