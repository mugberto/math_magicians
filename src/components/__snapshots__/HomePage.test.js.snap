// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correctly 1`] = `
<div
  className="container page-border"
>
  <h2>
    Welcome to our page
  </h2>
  <p>
    Lorem ipsum dolor, sit amet consectetur adipisicing elit. Doloremque minus similique impedit iste sunt est consequuntur ratione, aperiam nobis quidem. Quo sunt, quia sequi delectus, quaerat ipsa nam hic suscipit.
  </p>
  <p>
    Lorem ipsum dolor, sit amet consectetur adipisicing elit. Doloremque minus similique impedit iste sunt est consequuntur ratione, aperiam nobis quidem. Quo sunt, quia sequi delectus, quaerat ipsa nam hic suscipit.
  </p>
</div>
`;
